Wrong QMake selected on OSX

Following instructions [here](https://github.com/mantidproject/mantid/wiki/Build-environment-setup-on-OS-X-10.12-Sierra) followed by running cmake. cmake does not configure properly

`cmake {source_dir}`

```
CMake Error at /usr/local/Cellar/cmake/3.10.1/share/cmake/Modules/FindQt4.cmake:1320 (message):
 Found unsuitable Qt version "5.10.0" from /usr/local/opt/qt/bin/qmake, this
 code requires Qt 4.x
```

Setting `QT_QMAKE_EXECUTABLE` to `/usr/local/bin/qmake` provides a workaround.

--- a/buildconfig/CMake/DarwinSetup.cmake
+++ b/buildconfig/CMake/DarwinSetup.cmake
@@ -36,8 +36,8 @@ message (STATUS "Operating System: Mac OS X ${OSX_VERSION} (${OSX_CODENAME})")
 # Enable the use of the -isystem flag to mark headers in Third_Party as system headers
 set(CMAKE_INCLUDE_SYSTEM_FLAG_CXX "-isystem ")
 
-# Add default location of Qt5 from homebrew to prefix path so that find_package works
-list ( APPEND CMAKE_PREFIX_PATH "/usr/local/opt/qt" )
+# Set Qt5 dir according to homebrew location
+set ( Qt5_DIR /usr/local/opt/qt/lib/cmake/Qt5 )
 
 ###########################################################################
 # Use python libraries associated with PYTHON_EXECUTABLE

